index.html

```diff
@@ -1303,274 +1303,230 @@ <h2>Extensibility</h2>
+A <a>DID</a> is associated with a <a>DID document</a>.
+<a>DID documents</a> are expressed using the
+<a href="#data-model">data model</a> and can be serialized into a
+<a href="#representations">representation</a>.
 The following sections define the properties in a <a>DID document</a>,
 including whether these properties are required or optional. These properties
 describe relationships between the <a>DID subject</a> and the value of the
 property.
     </p>
-
     <p>
-This specification defines three classes of core properties:
-    </p>
-
-    <ol>
-      <li>
-properties defined for a <a>DID Document</a>, the topmost
-<a data-cite="INFRA#ordered-map">map</a> in the <a href="#data-model">data
-model</a>;
-      </li>
-      <li>
-properties defined for a <a>Verification Method</a>, a concept
-represented by a <a data-cite="INFRA#ordered-map">map</a> in the
-<a href="#data-model">data model</a>, and specified in
-Section <a href="#verification-methods"></a>;
-      </li>
-      <li>
-properties defined for a <a>Service</a>, a concept
-represented by a <a data-cite="INFRA#ordered-map">map</a> in the
-<a href="#data-model">data model</a>, and specified in Section
-<a href="#services"></a>.
-      </li>
-    </ol>
-
-    <p>
-The core properties are summarized informatively in Section
-<a href="#summary"></a>, and described in detail and specified
-normatively in subsequent sections.
-    </p>
-
-
-    <p class="note" title="Ordering of values">
-As a result of the <a href="#data-model">data model</a> being defined using
-terminology from [[INFRA]], property values which can contain more than one
-item, such as <a data-cite="INFRA#ordered-map">maps</a> and
-<a data-cite="INFRA#ordered-set">sets</a>, are explicitly ordered. For the
-purposes of this specification, unless otherwise stated, ordering is not
-important and implementations are not expected to produce or consume
-deterministically ordered values.
-    </p>
-
-    <section class="informative">
-      <h2>Summary</h2>
-
-      <p>
-This section contains an informative reference for the core properties defined
-by this specification, with expected values and whether or not they are
+The following tables contain informative references for the core properties
+defined by this specification, with expected values, and whether or not they are
 required. The property names in the tables are linked to the normative
 definitions and more detailed descriptions of each property.
-      </p>
-      <p class="note" title="Property names used in multiple classes">
-  The property names <code>id</code>, <code>type</code>, and
-  <code>controller</code> can be present in different classes with possible
-  differences in constraints.
-      </p>
+    </p>
+    <p class="note" title="Property names used in multiple classes">
+The property names <code>id</code>, <code>type</code>, and
+<code>controller</code> can be present in different classes with possible
+differences in constraints.
+    </p>
```

Comment:
```suggestion
    </p>
    <p class="note" title="Property names used in maps of different types">
The property names <code>id</code>, <code>type</code>, and
<code>controller</code> can be present in maps of different types
with possible differences in constraints.
    </p>
```

With the "three classes of core properties" text removed, "classes" here no longer reads well.